Makefile

```diff
@@ -47,7 +47,7 @@ setup_env:
 	fi
 
 	echo "Installing $(CONCRETE_PYTHON_VERSION)" && \
-	poetry run python -m pip install -U --pre "$(CONCRETE_PYTHON_VERSION)" --no-deps
+	poetry run python -m pip install -U --pre "$(CONCRETE_PYTHON_VERSION)"
```

Comment:
there was not a reason for us to have `--no-deps`?

Reply:
weird but ok

Reply:
ok interesting then 1